.github/workflows/win.yml

```diff
@@ -376,3 +376,188 @@ jobs:
+            dlls/mpir.dll
+            dlls/yaml.dll
+            dlls/libxml2.dll
+          key: win-dlls-${{ hashFiles('.github/workflows/win.yml', 'etc/win-ci/*.ps1') }}-msvc
+          fail-on-cache-miss: true
+      - name: Restore OpenSSL DLLs
+        uses: actions/cache/restore@v3
+        with:
+          path: |
+            libs/crypto-dynamic.lib
+            libs/ssl-dynamic.lib
+            dlls/libcrypto-3-x64.dll
+            dlls/libssl-3-x64.dll
+          key: win-openssl-dlls-3.1.0-${{ hashFiles('etc/win-ci/build-openssl.ps1') }}-msvc
+          fail-on-cache-miss: true
+      - name: Restore LLVM
+        uses: actions/cache/restore@v3
+        with:
+          path: llvm
+          key: llvm-libs-16.0.3-msvc
+          fail-on-cache-miss: true
+
+      - name: Set up environment
+        run: |
+          echo "CRYSTAL_LIBRARY_PATH=$(pwd)\libs" >> ${env:GITHUB_ENV}
+          echo "LLVM_CONFIG=$(pwd)\llvm\bin\llvm-config.exe" >> ${env:GITHUB_ENV}
+
+      - name: Build LLVM extensions
+        run: make -f Makefile.win deps
+
+      - name: Build Crystal
+        run: |
+          bin/crystal.bat env
+          make -f Makefile.win -B release=1
+
+      - name: Download shards release
+        uses: actions/checkout@v3
+        with:
+          repository: crystal-lang/shards
+          ref: v0.17.3
+          path: shards
+
+      - name: Download molinillo release
+        uses: actions/checkout@v3
+        with:
+          repository: crystal-lang/crystal-molinillo
+          ref: v0.2.0
+          path: shards/lib/molinillo
+
+      - name: Build shards release
+        working-directory: ./shards
+        run: ../bin/crystal.bat build --release src/shards.cr
+
+      - name: Gather Crystal binaries
+        run: |
+          mkdir crystal/src
+          mkdir crystal/lib
+          cp .build/crystal.exe crystal/
+          cp .build/crystal.pdb crystal/
+          cp shards/shards.exe crystal/
+          cp libs/* crystal/lib/
+          cp dlls/* crystal/
+          cp src/* crystal/src -Recurse
+          cp README.md crystal/
+          cp LICENSE crystal/LICENSE.txt
+          rm crystal/src/llvm/ext/llvm_ext.obj
+
+      - name: Upload Crystal binaries
+        uses: actions/upload-artifact@v3
+        with:
+          name: crystal-release
+          path: crystal
+
+  x86_64-windows-installer:
+    runs-on: windows-2022
+    needs: [x86_64-windows-release]
+    steps:
+      - name: Disable CRLF line ending substitution
+        run: |
+          git config --global core.autocrlf false
+
+      - name: Download Crystal source
+        uses: actions/checkout@v3
+
+      - name: Download Crystal executable
+        uses: actions/download-artifact@v3
+        with:
+          name: crystal-release
+          path: etc/win-ci/portable
+
+      - name: Restore LLVM
+        uses: actions/cache/restore@v3
+        with:
+          path: llvm
+          key: llvm-libs-16.0.3-msvc
+          fail-on-cache-miss: true
```

Comment:
Why does this job require LLVM?

Reply:
The standard library docs include LLVM libraries, which require llvm-config